packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -186,8 +185,6 @@ def test_signal_service(dcos_api_session):
     all_slave_units = [
         'docker-gc-service',
         'docker-gc-timer',
-        'metrics-agent-service',
-        'metrics-agent-socket',
```

Comment:
Won't we need the telegraf service on the agent nodes too?
Also, add a TODO for the telegraf socket?